part2/stages/Commit-preserved-data

```diff
@@ -31,6 +31,7 @@ done
 . "${INSTALL_MODE_CONF}"
 
 if [ "${INSTALL_MODE}" = "fresh" ] ; then
+    . /usr/lib/openxt/key-functions
```

Comment:
Is this needed?  Was it broken before?

Reply:
Yes, oddly it was failing during our tests without this.